src/js/views/eva-variant-view.js

```diff
@@ -193,7 +193,8 @@ EvaVariantView.prototype = {
 
     // For a given RS ID, get associated SS ID
     getAssociatedSSIDsFromAccessioningService: function(accessionCategory, accessionID) {
-         return this.getAccessioningWebServiceResponse(accessionCategory, accessionID.substring(2) + "/submitted");
+        var response = this.getAccessioningWebServiceResponse(accessionCategory, accessionID.substring(2) + "/submitted");
```

Comment:
Given that right now we don't have the SS IDs imported for human, can you validate if the query is for human and not call the endpoint?
Please check if this applies in the multi-view as well

Reply:
I didn't want to put that time bomb in there just in case we forget to update the code after the human SS IDs are imported. So I just made the result resilient to errors when calling the endpoint.